main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -28,7 +27,20 @@ sealed trait Completion {
     */
   def toCompletionItem: CompletionItem = this match {
     case Completion.KeywordCompletion(name, context) =>
-      CompletionItem(label = name, sortText = Priority.normal(name), textEdit = TextEdit(context.range, s"$name "), kind = CompletionItemKind.Keyword)
+      CompletionItem(label = name, sortText = Priority.normal(name), textEdit = TextEdit(context.range, s"$name "),
+        kind = CompletionItemKind.Keyword)
+    case Completion.FieldCompletion(name, context) =>
+      CompletionItem(label = name, sortText = Priority.high(name), textEdit = TextEdit(context.range, s"$name "),
+        kind = CompletionItemKind.Field)
+    case Completion.PredicateCompletion(name, priority, context) =>
+      CompletionItem(label = name, sortText = priority, textEdit = TextEdit(context.range, s"$name "),
+        kind = CompletionItemKind.Variable) // Variable?
+    case Completion.BuiltinTypeCompletion(name, priority, textEdit, insertTextFormat) =>
+      CompletionItem(label = name, sortText = priority, textEdit = textEdit, insertTextFormat = insertTextFormat,
+        kind = CompletionItemKind.Enum) //Enum?
```

Comment:
Here you should just whatever kind that we chosen previously. Dont worry too much about it, just preserve the information that was already there.